interface-definitions/include/ldp-sync.xml.i

```diff
@@ -0,0 +1,27 @@
+<!-- include start from ldp-igp-sync.xml.i -->
+<node name="ldp-sync">
+  <properties>
+    <help>LDP-IGP synchronization configuration for interface</help>
+  </properties>
+  <children>
+    <leafNode name="disable">
+      <properties>
+        <help>Disable LDP-IGP synchronization for interface</help>
+        <valueless/>
+      </properties>
+    </leafNode>
+    <leafNode name="holddown">
```

Comment:
The `holddown` CLI node is defined multiple times - please move it to an include file and reference it.

Reply:
So this is something that I believe we *can* do but in FRR it's kind of split.

So the reason I did it the way I did it was because the way LDP-IGP sync is done is it's done in two places. Under the routing protocol (for global enablement) and then under the interface. By default from what I read/understood is that we are to enable/disable it under the protocol and it will propagate it down to the interfaces. Now the interfaces in *theory* have 2 (and it seems 3, but this is where it seems to be buggy) options. One option is to disable LDP-IGP sync for that interface (the disable) and one is to change the `holddown` value for just that interface. I figured to make a template of the interfaces and include that, but to make the protocol portion be independent from the interfaces.

If you're thinking we can consolidate them then can I do a protocol LDP-IGP sync include file, and an interfaces include file? That way if there's changes in the global it doesn't affect the interfaces. Vice versa with the interfaces affecting the global.